src/main/java/frc/robot/subsystems/drive/PhoenixDrive.java

```diff
@@ -224,6 +274,7 @@ public Command sysIdDynamic(SysIdRoutine.Direction direction) {
 
     @Override
     public void periodic() {
+        this.setAligning(true);
```

Comment:
This probably should be set via a command rather than always forcing alignment

Reply:
that was for testing, updated now